subsys/bluetooth/controller/ll_sw/ull_llcp_remote.c

```diff
@@ -428,7 +449,7 @@ static void rr_st_idle(struct ll_conn *conn, uint8_t evt, void *param)
 				/* Run remote procedure */
 				rr_act_run(conn);
 				rr_set_state(conn, RR_STATE_TERMINATE);
-			} else if (incompat == INCOMPAT_NO_COLLISION) {
+			} else if (!with_instant || incompat == INCOMPAT_NO_COLLISION) {
```

Comment:
Isn't this change unrelated to the procedure pause and belongs in a separate commit regarding procedure collision?

Reply:
yes it does - thank you!